api-gateway-config/scripts/lua/routing.lua

```diff
@@ -59,10 +59,15 @@ function _M.processCall()
       end
       -- Parse backend url
       local u = url.parse(opFields.backendUrl)
+      -- add http:// if no protocol is specified
+      if u.host == nil then
```

Comment:
It looks like host is the domain, where scheme is the protocol. So wouldn't we need to look for scheme here?

https://github.com/golgote/neturl#url-parser